### DISCUSSION TITLE
Architect Role

### DISCUSSION THREAD

**Discussion by OrlinVasilev** [vmware-tanzu/velero#5531] [General discussions]
Edited the title on the discussion because it can get out of the scope quite quickly only to "Architect role"

Hi All,

as of now @dsu-igeek is listed as Architect but as Dave shared he is not able to do that any more.
https://github.com/vmware-tanzu/velero/blob/main/MAINTAINERS.md?plain=1#L33 

So we have to decide if we will continue that role and find replacement from the community?
IMO technical lead role is more than enough to drive the technical discussion in the future and as  of now @reasonerjt is leading that effort GREAT! :)

Please share your thoughts below so we can move on with this one!

BR,

Orlix
Velero Community Manager

**Comment by shawn-hurley** [comment]
@OrlinVasilev it might help to understand what responsibilities the architect had vs. a tech lead in the past. 

The big question in my mind is, was there a separation of concerns that were put in place for a reason such that we can determine if it is still needed? 

I am unable to find anything in the project docs, so wondering if I am just not searching in the correct place :)

**Reply by OrlinVasilev** [reply]
Yup tagged Dave and Daniel if they can provide more information about it

**Comment by sseago** [comment]
I would also say that before we do anything in terms of suggesting a new architect and/or retiring the role that we would update the governance doc to provide a process by which these two roles (or one, if we decide we want to eliminate one) are assigned.

Also, it might be worth reaching out to people who have held this role in the past when the roles were split to understand how the roles differed in practice. At one point Dave was the tech lead and Nolan was the architect, and then there was a point where they swapped places. I wasn't involved enough at the time to really understand what the distinction was in practice, though.

**Reply by kaovilai** [reply]
@nrb can you comment?

**Reply by OrlinVasilev** [reply]
Absolutely! That's the discussion about to decide what to do and how to do it :)

**Comment by OrlinVasilev** [comment]
@dsu-igeek and @reasonerjt - can you please provide your input here :) thanks!

**Reply by reasonerjt** [reply]
@OrlinVasilev 
I personally don't think the architect is a must-have role for velero.

Based on my experience in open-source projects, the technical decisions about the project can be made via discussion around maintainers.

**Comment by OrlinVasilev** [comment]
Edited the title on the discussion because it can get out of the scope quite quickly only to "Architect role"

**Comment by OrlinVasilev** [comment]
Dave was internal VMware architect behind Velero(Tanzu backup strategy) at the time! As of now the project is led by the community so that internal architect role is not relevant to the project in the same terms and we can just remove it from the https://github.com/vmware-tanzu/velero/blob/main/MAINTAINERS.md?plain=1#L33 as well as to move @dsu-igeek to emeritus!
